lib/deck.rb

```diff
@@ -0,0 +1,24 @@
+class Deck
+  attr_reader :cards
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.size
+  end
+
+  def cards_in_category(thing)
+    card_category = []
+    cards.each do |card|
+      if card.category == thing
+        card_category << card
+      else
```

Comment:
no need for an `else` if there is not else condition that you need to take care of.